modules/nw-exposing-service.adoc

```diff
@@ -106,6 +116,22 @@ nodejs-ex-ingress   NodePort    172.30.107.72    <none>        3306:31345/TCP
 ----
 $ oc delete svc nodejs-ex
 ----
+
+.Verification
+
+* Verify that the service node port is updated with a port in the `30000-32767` range. For example, `30327`:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
+[source,terminal]
+----
+NAME    TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+httpd   NodePort   172.xx.xx.xx    <none>        8443:30327/TCP   109s
+----
```

Comment:
Suggestion that includes the "enter the following command" language:
```suggestion
* To check that the service node port is updated with a port in the `30000-32767` range, enter the following command:
+
[source,terminal]
----
$ oc get svc
----
+
In the following example output, the updated port is `30327`:
+
.Example output
[source,terminal]
----
NAME    TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
httpd   NodePort   172.xx.xx.xx    <none>        8443:30327/TCP   109s
----
```
Feel free to spike it.